python/cudf/cudf/core/dataframe.py

```diff
@@ -269,6 +269,11 @@ def _init_from_list_like(self, data, index=None, columns=None):
         self._index = as_index(index)
         data = list(itertools.zip_longest(*data))
 
+        if columns is not None and len(data) == 0:
+            data = [
+                cudf.core.column.column_empty(row_count=0, dtype=None)
+            ] * len(columns)
```

Comment:
This will create a list of `len(columns)` where each element in the list is a reference to the same column object which seems unintended. Should make this a list comprehension instead so we properly create `len(columns)` column objects.

Reply:
I see, yeah thanks for catching this - Made the change.

```python
>>> import cudf
>>> s = [cudf.Series([1, 2, 3])] * 5
>>> s
[0    1
1    2
2    3
dtype: int64, 0    1
1    2
2    3
dtype: int64, 0    1
1    2
2    3
dtype: int64, 0    1
1    2
2    3
dtype: int64, 0    1
1    2
2    3
dtype: int64]
>>> s[0]
0    1
1    2
2    3
dtype: int64
>>> s[0].name = "abc"
>>> s
[0    1
1    2
2    3
Name: abc, dtype: int64, 0    1
1    2
2    3
Name: abc, dtype: int64, 0    1
1    2
2    3
Name: abc, dtype: int64, 0    1
1    2
2    3
Name: abc, dtype: int64, 0    1
1    2
2    3
Name: abc, dtype: int64]